libraries/chain/steem_evaluator.cpp

```diff
@@ -656,14 +722,26 @@ namespace golos { namespace chain {
                             com.cashout_time = fc::time_point_sec::maximum();
                         }
 
-                        if (_db.has_hardfork( STEEMIT_HARDFORK_0_17__431)) {
+                        if (_db.has_hardfork(STEEMIT_HARDFORK_0_17__431)) {
                             com.cashout_time = com.created + STEEMIT_CASHOUT_WINDOW_SECONDS;
                         }
 
+                        if (_db.has_hardfork(STEEMIT_HARDFORK_0_19__295)
```

Comment:
Not needed case.
Referrer appears on create of account, it can't appear after some time.
So, additing of beneficiary can happen only on create post/comment.